components/image-placeholder/ImageWithPlaceholder.tsx

```diff
@@ -0,0 +1,42 @@
+'use client';
+import React from 'react';
+import { PostImagePlaceholder } from '@/components/placeholders/PostImagePlaceholder';
+import Image from 'next/image';
+
+interface IProps {
+  src: string;
+  alt: string;
+}
+
+const ImageWithPlaceholder = ({ src, alt }: IProps) => {
```

Comment:
I would name the component the same as the directory:
ImageWithPlaceholder.tsx --> image-with-placeholder or
ImagePlaceholder.tsx --> image-placeholder

Reply:
yeah, i was thinking about the same. forgot it. Good Catch!